problem-3/blogs/apps.py

```diff
@@ -0,0 +1,6 @@
+from django.apps import AppConfig
+
+
+class BlogsConfig(AppConfig):
```

Comment:
app 명은 보통 plural case를 사용하지 않는 것을 추천드려요. 셀 수 없는 app 명도 많기 때문(가령, auth 등)

```suggestion python
class BlogConfig(AppConfig):
```